package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@pxblue/icons",
-    "version": "1.4.0-beta.0",
+    "version": "1.4.0",
```

Comment:
Did you try using the new beta that was published to make sure it still works as expected (maybe in the Angular storybook to also see if the Hero With Icons and Sizes story looks better)?

Reply:
Can verify
![image](https://user-images.githubusercontent.com/8997218/125628309-e7651d06-d5fa-4b93-863f-1f082d7f7fe1.png)
![image](https://user-images.githubusercontent.com/8997218/125628442-29e9e15e-a85b-4507-aeeb-af92966dfcca.png)

Reply:
current still looks a little janky, but it's definitely no worse than it was before